README.md

```diff
@@ -6,59 +6,119 @@ DUS leverages the power of Amazon Textract, Amazon Comprehend , Amazon Comprehen
 
 ## Architecture Diagram
 
-
 ![img](./images/DUS_Arch.png)
 
+Current document formats supported : **PDF,JPG,PNG**
 
+## 2. CICD Deploy
```

Comment:
Could you number the CICD and Dev Deploy in the right order?

Reply:
updated